src/modules/simulator/simulator.cpp

```diff
@@ -166,11 +126,6 @@ int Simulator::start(int argc, char *argv[])
 			_instance->poll_for_MAVLink_messages();
 #endif
 
-		} else if (argv[2][1] == 'p') {
```

Comment:
I think this breaks HITL for Raspberry Pi or something like that but I'm not sure anymore.

Reply:
Shouldn't matter anymore without the need for most of the simulator sensors.